src/main/resources/hudson/plugins/ansicolor/Messages.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-DisplayName = Color ANSI Console Output
+plugin_DisplayName=Color ANSI Console Output

```

Comment:
My understanding is that it will not change anything at runtime. It is used at build time for checking for bugs - the annotation has `@Retention(RetentionPolicy.CLASS)` so I don't think there would be any way for runtime to have any different behavior. I think it is included by the newer parent plugin version which is why the build fails with.

If I run the `mvn verify` with the suppressions removed I get:

```
[INFO] ------------------------------------------------------------------------                                        
[INFO] BUILD FAILURE                                       
[INFO] ------------------------------------------------------------------------                                        
[INFO] Total time: 49.565 s                                
[INFO] Finished at: 2017-07-20T10:18:59-05:00              
[INFO] Final Memory: 63M/1070M                             
[INFO] ------------------------------------------------------------------------                                        
[ERROR] Failed to execute goal org.codehaus.mojo:findbugs-maven-plugin:3.0.4:check (findbugs) on project ansicolor: failed with 3 bugs and 0 errors  -> [Help 1]                                                                              
[ERROR]                                                    
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.                                    
[ERROR] Re-run Maven using the -X switch to enable full debug logging.                                                 
[ERROR]                                                    
[ERROR] For more information about the errors and possible solutions, please read the following articles:              
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

I see a way in the parent plugin POM to configure exclude rules at `${project.basedir}/src/findbugs/excludesFilter.xml` which may be an approach for a few of these, but I'm not sure.